UnityProject/Assets/Scripts/Items/Medical/DefibrillatorPaddles.cs

```diff
@@ -98,12 +99,12 @@ void Perform()
 		bar.ServerStartProgress(interaction.Performer.RegisterTile(), Time, interaction.Performer);
 	}
 
-	private async void Cooldown()
+	private async void Cooldown(Vector3 soundPos)
 	{
 		onCooldown = true;
 		await Task.Delay(cooldownMs).ConfigureAwait(false);
 		onCooldown = false;
-		SoundManager.PlayNetworkedAtPos(soundCharged, gameObject.AssumedWorldPosServer());
+		SoundManager.PlayNetworkedAtPos(soundCharged, soundPos);
```

Comment:
wouldn't this be weird on a moving shuttle?

Reply:
You're right so i fixed this